### DISCUSSION TITLE
Trying to install but stuck with error

### DISCUSSION THREAD

**Discussion by sunilsingh2019** [vercel/next.js#40459] [Help]
**This error only showing when i run with local wordpress setup but on live server working fine **
Cloned from : https://github.com/vercel/next.js/tree/canary/examples/cms-wordpress
installed node v: v16.17.0
when i run this command 'npm run dev' getting following error:
error - Failed to load next.config.js, see more info here https://nextjs.org/docs/messages/next-config-error
TypeError: Cannot read properties of null (reading '0')
    at Object.<anonymous> (/Users/Webee/Desktop/cms-wordpress-app/next.config.js:12:72)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:170:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:533:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:438:15)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**local.env file**

```
WORDPRESS_API_URL=http://localhost/cms-wordpress-app/graphql
# Only required if you want to enable preview mode
# WORDPRESS_AUTH_REFRESH_TOKEN=
# WORDPRESS_PREVIEW_SECRET=
```

**config.next.js**
```
if (!process.env.WORDPRESS_API_URL) {
  throw new Error(`
    Please provide a valid WordPress instance URL.
    Add to your environment variables WORDPRESS_API_URL.
  `)
}


/** @type {import('next').NextConfig} */
module.exports = {
  future: {
    webpack5: true,
  },
  images: {
    domains: [
      process.env.WORDPRESS_API_URL.match(/(?!(w+)\.)\w*(?:\w+\.)+\w+/)[0], // Valid WP Image domain.
      '0.gravatar.com',
      '1.gravatar.com',
      '2.gravatar.com',
      'secure.gravatar.com',
    ],
  },
}
```

**Comment by sunilsingh2019** [comment]
I have fixed this issue followed that steps https://stackoverflow.com/questions/65519568/cannot-find-module-caniuse-lite-dist-unpacker-agents-when-running-create-react

**Comment by Forseti93** [comment]
That error starts on 12 row of the next.config.js. See error message: "...at Object. (/Users/Webee/Desktop/cms-wordpress-app/next.config.js:12:72)..."

Just delete rwgEx in next.config.js
was      
      process.env.WORDPRESS_API_URL.match(/(?!(w+)\.)\w*(?:\w+\.)+\w+/)[0], // Valid WP Image domain.
now      
      // process.env.WORDPRESS_API_URL.match(/(?!(w+)\.)\w*(?:\w+\.)+\w+/)[0], // Valid WP Image domain.
      process.env.WORDPRESS_API_URL.match()[0], // Valid WP Image domain.

Minus is, that I don't understand what we are importing with that string. Someone know?
